yarn-project/end-to-end/src/e2e_card_game.test.ts

```diff
@@ -65,7 +65,7 @@ describe('e2e_card_game', () => {
     // Card stats are derived from the users' private keys, so to get consistent values, we set up the
     // initial sandbox accounts that always use the same private keys, instead of random ones.
     ({ pxe, logger, teardown } = await setup(0));
-    wallets = await Promise.all((await deployInitialSandboxAccounts(pxe)).map(a => a.account.getWallet()));
+    wallets = await getSandboxAccountsWallets(pxe);
```

Comment:
Just checking. People can still run the e2e tests without the sandbox can't they? Previously if you didn't specify the `PXE_URL` then it would create a local setup. Is that still possible?

This looks like it forces the test to use sandbox accounts.

Reply:
Ah that's a good point. Most will still work, some that require slightly different methods (like here) will need conditional checks. Will fix

Reply:
fixed in https://github.com/AztecProtocol/aztec-packages/pull/2891/commits/f4c69e42f258b7c2011dc9055e112a9cb4f7ffc1